assets/js/hooks/infinite_scroll.js

```
@@ -0,0 +1,74 @@
+/**
+ * Sprint 15A: Infinite scroll hook for seamless pagination.
+ * 
+ * Automatically triggers load more events when user scrolls near
+ * the bottom of the container, providing a smooth UX for large datasets.
+ */
+
+const InfiniteScroll = {
+  mounted() {
+    this.threshold = parseInt(this.el.dataset.threshold) || 200;
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)